hms-app/src/actions/HackathonActions.tsx

```diff
@@ -28,21 +28,45 @@ export const getHackathonDetails = (hackathonID: string) => {
     .catch((err) => console.log(err))
 }
 
-export const createHackathon = (props: {
-  endDate: string
-  title: string
-  startDate: string
-}) => {
+export const createHackathon = (
+  hackathonText: string,
```

Comment:
hackathonText is a bit vague, hackathonTitle or just title is a bit more descriptive